src/components/dev-hub/byline-block.js

```diff
@@ -65,14 +66,14 @@ const ByLine = styled('div')`
     }
 `;
 
-const BylineBlock = ({ author, authorImage }) => {
+const BylineBlock = ({ authorName, authorImage }) => {
     return (
         <ByLine>
             <AuthorImage>
-                <img src={authorImage} alt={author} />
+                <img src={withPrefix(authorImage)} alt={authorName} />
             </AuthorImage>
             <AuthorText collapse>
-                By <AuthorLink to="#">{author}</AuthorLink>
+                By <AuthorLink to="#">{authorName}</AuthorLink>
```

Comment:
Can we populate this with the planned url (`/author/<authorName>`)?

We'll need to do something like authorName.split(' ').join('-') or an equivalent regex plus `encodeURIComponent`